cmd/status/printers/event/printer.go

```diff
@@ -52,25 +63,27 @@ func (ep *Printer) Print(ch <-chan pollevent.Event, identifiers object.ObjMetada
 	return err
 }
 
-func (ep *Printer) printStatusEvent(se pollevent.Event) {
+func (ep *Printer) printStatusEvent(se pollevent.Event) error {
 	switch se.Type {
 	case pollevent.ResourceUpdateEvent:
 		id := se.Resource.Identifier
-		printResourceStatus(id, se, ep.IOStreams)
+		var invName string
+		var ok bool
```

Comment:
Do we need to define these here? It seems like we can just use the `:=` assignment in the if statements.

Reply:
@mortent  For now since the invName variable is used outside of the if statement so it cannot be accessed if using the `:=` assignment in the if statement. I could also add an else statement and then move the following code into the else statement to get rid of the definitions outside, but I am not sure if having an else statement followed by an if statement that leads to return is a great idea or not. Do you have any suggestions on that?

Reply:
Yeah, you are correct. I didn't notice the `invName` was used outside of the if statement